### DISCUSSION TITLE
RealityModel can not show complete

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#3071] [Q&A]
now ,i have the RealityModel (3dtiles ),and can show the road in cesium,but can not show the road in itwinjs(package version 2.19.28)
thw attach pic : 
![1](https://user-images.githubusercontent.com/18022720/150323324-4391d119-84ca-49b7-84d5-bf555c061ed6.png)
![2](https://user-images.githubusercontent.com/18022720/150323336-55cbacd1-8d1f-4edc-b994-c32cd5e57802.png)

**Comment by pmconne** [comment]
If you can supply the tileset please do so. That will make it much, much easier to assist you.
Otherwise:
- Check browser console and network tab for any errors.
- Check [IModelApp.tileAdmin.statistics.totalFailedRequests](https://www.itwinjs.org/reference/imodeljs-frontend/tiles/tileadmin.statistics/). If it's greater than zero we may be failing to obtain/read those tiles.
- Try upgrading to the iTwin.js 3.0 release candidate. It includes numerous fixes to the glTF reader, which may address problems reading those tiles.
